src/harness/testcases/WINNF_FT_S_SSS_testcase.py

```diff
@@ -79,12 +85,15 @@ def test_WINNF_FT_S_SSS_8(self, config_filename):
     Checks that SAS UUT response with fatal alert message.
     """
     config = loadConfig(config_filename)
+    certificate_hash = getCertificateFingerprint(config['sasCert'])
+    self._sas_admin.InjectPeerSas({'certificateHash': certificate_hash,
```

Comment:
In all cases where you inject the peer SAS, you need to first call Reset() since injecting the peer SAS will alter the state of the SAS UUT.

Reply:
As per   the suggestion admin Reset is called in all the test cases which have call to   injectPeerSAS API     self.sasReset used which internally calls the self._sas_admin.Reset()